SEP-0002-revised.md

```diff
@@ -0,0 +1,262 @@
+
+* A good grasp of the philosophy of the SunPy Project
+* A solid track record of being constructive and helpful
+* Significant contributions to the Project's goals, in any form
+* A willingness to dedicate time to improving the Project
+* A willingness to recruit, train, and nominate new team members
+* A track record of adhering to the project Code of Conduct.
+
+Project Membership acknowledges sustained and valuable efforts that align well with the philosophy and the goals of the SunPy Project.
+
+#### Initial Membership
+
+To be able to elect new Project Members an initial set of Project Members are needed.
+People meeting the following criteria will be invited to form the initial group of Project Members.
+They should consider the above responsibilities and actively agree to taking them on if they choose to accept the invitation.
+
+* Current commit access to the core repo, or any accepted affiliated package maintained by the project, and with a commit to that repository within the last 12 months.
+* Currently holds a role listed under the community roles section of the website: <https://sunpy.org/project/##community-roles>.
+* Is a currently serving Advisory Board member.
+
+#### Adding New Project Members
+
+Anyone is eligible to be nominated to be a Project member, as long as they meet the criteria laid out in the Membership section above.
+Self-nominations are encouraged.
+
+The process for adding a new Project member is as follows:
+
+1. A nomination is received by the Steering Committee, a nomination should include a short, factual, statement stating how the nominee meets the criteria for Project Membership.
+The Steering Committee, and the Ombudsperson check that the nomination is factually accurate and their record of contribution to the community adheres to the Code of Conduct.
+1. The candidate is then put to the Project Members.
+The name and nomination statement of the candidate will be shared with the Project Members.
+The candidate can become a Project Member if, during the two weeks following the candidate being put to the Project Members, five existing Project members approve the candidate and no Project Members call for an election.
+1. Project Members can call for an election either by informing the Steering Committee or the Ombudsperson.
+If a vote is called for then the candidate will be come a Project Member with a simple majority of all votes cast in the election.
+An election needs to be announced to the Project Members one week before the voting opens and voting will be open for two weeks.
+1. The candidate will be informed promptly of the outcome by the Steering Committee.
+
+The name of a candidate should not be shared outside of the Project Members until they become a Project Member.
+A public list of Project Members will be kept on the SunPy website.
+
+Nominations, approvals and any elections of Project Members should take place before the initial election of the Steering Committee.
```

Comment:
What does this mean? Can't elections of Project Members occur at any time?

Reply:
Yes, this is specifically about the first set of elections. To phrase it a slightly different way it is saying "the initial set of project members must hold an election for more project members before electing the first steering committee".